media-plugins/gst-plugins-qt5/gst-plugins-qt5-1.18.4.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+GST_ORG_MODULE=gst-plugins-good
+
+inherit gstreamer-meson
+
+DESCRIPTION="Qt5 QML video sink plugin for GStreamer"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
```

Comment:
did you test all those arches?

Reply:
I haven't tested the ebuild on all of those architectures, but I have built that specific plugin on those platforms and used it. Would it be best to just have "~amd64" in that list and leave the others out until someone tests the ebuild on those platforms?

Reply:
I have removed everything but amd64 for now